README.md

```diff
@@ -24,7 +24,7 @@ sections below if you'd like to help.
 
 ## Setting up Swap My Vote locally
 
-You will need Ruby installed.
+You will need Ruby and [Yarn](https://yarnpkg.com/lang/en/docs/install/) installed.
```

Comment:
Let's explicitly mention the requirement on node 10 if that's the case.  Seems that we also need to trigger a `yarn install` somehow e.g. `rails yarn:install`?